### PR TITLE
Fixed Dimens and Redundant Res

### LINKED ISSUE #453: Remove duplicate resources and dimens values
**Describe the bug**
Some resource files are redundant in the app codebase and not needed. 
Also, there are duplicate dimens values. Remove them and be sure to make changes wherever the duplicates were used throughout the codebase.

**Expected Behavior**
Remove duplicate resources and dimens values

**Steps to reproduce it**
Go through the resources and codebase of the app.

**Would you like to work on the issue?**
Issue for Codeheat.

### PR DESCRIPTION
**Pull request by CodeMonk263**
Fixes #453 

**Changes**: Chaned multiple xml files which contained redundant values and deleted res files which were not being used.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by CodeMonk263** [comment]
Can someone review this PR

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
     <dimen name="padding_large">16dp</dimen>
-    <dimen name="padding_intermediate">24dp</dimen>
+<!--    <dimen name="padding_intermediate">24dp</dimen>-->
     <dimen name="padding_extra_large">32dp</dimen>
     <dimen name="padding_extra_larger">64dp</dimen>
-    <dimen name="padding_mem_graph">54dp</dimen>
+<!--    <dimen name="padding_mem_graph">54dp</dimen>-->
     <dimen name="margin_large">60dp</dimen>
-    <dimen name="margin_extra_large">248dp</dimen>
+<!--    <dimen name="margin_extra_large">248dp</dimen>-->
```
Comment (jddeep) [inline app/src/main/res/values/dimens.xml]: Removing unused resources doesn't mean commenting them out. Please check well if a `res` is used anywhere in a redundant way i.e, the same resource value is kept with different names, remove them. Also be sure to change in the codebase/XML files where those removed resources were used.

**Review by jddeep — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (CodeMonk263) [reply app/src/main/res/values/dimens.xml]: Ok thanks i'll update soon

Comment (CodeMonk263) [reply app/src/main/res/values/dimens.xml]: There are some unused declarations. I'm confused if i should remove them or they are for future use.

Comment (jddeep) [reply app/src/main/res/values/dimens.xml]: If they are unused, just remove them. In future, we will create if needed.

**Review by jddeep — CHANGES_REQUESTED** [review]
Please squash your commits into one.

**Pull request closed without merging**